Gemfile

```diff
@@ -20,6 +20,8 @@ gem 'rails', '>0.a'
 # Provides basic authentication functionality for testing parts of your engine
 gem 'solidus_auth_devise'
 
+gem 'omniauth-rails_csrf_protection'
```

Comment:
This needs to be added to the `gemspec` file. Otherwise, it won't be pulled when users have `solidus_social` as a dependency.

Reply:
To be clear, it needs to be in the `gemspec` and it's not needed in the `Gemfile`.

Reply:
It doesn't work when it's in the `gemspec` somehow
Specs fail when it is then `gemspec`, so that's why I moved it to the Gemfile.

Maybe we have to keep it in both the `Gemfile` and the `gemspec` so that it works for both the specs on the extension as well as for users that use `solidus_social` as a dependency.

What do you think ?

Reply:
That's weird. Which error are you getting?

Reply:
Sign in fails for the providers with the message that "Could not authenticate from (Provider) because it is forbidden".

Reply:
Added `omniauth-rails_csrf_protection` to the gemspec and modified the `engine.rb` file to add `require 'omniauth/rails_csrf_protection'` to allow specs to run.